### DISCUSSION TITLE
[Question]: Is it possible to customize  detections and labels for different zones?

### DISCUSSION THREAD

**Discussion by lee-costa** [blakeblackshear/frigate#12960] [Ask A Question] [answered]
### What is your question:

My configuration:

    review:
      alerts:
        labels: []
      detections:
        labels:
          - car
          - people
          - dog
          - cat
          - bicycle
        required_zones:
          - street
          - driveway

If I understand correctly, the above will only only create detections for the labels car, people, dog, cat, bicycle. Is it possible to have a different type of labels for the street and a different set for driveway under the same camera?

Something like:

    review:
      alerts:
        labels: []
      detections:
        labels:
         # - car
          - people
          - dog
          - cat
          - bicycle
        required_zones:
          - driveway
        labels:
          - car
          - people
          - dog
          - cat
          - bicycle
        required_zones:
          - street

**Comment by NickM-27** [comment, marked as answer]
You can set labels for each zone that limits which labels are relevant for that zone.
